### ISSUE TITLE
[OSC] If a user has previously used a port in the session it is not usable by other users

### ISSUE THREAD

**Issue by epicEaston197** [Yellow-Dog-Man/Resonite-Issues#1892]
### Describe the bug?

If a user had previously used a port it is not usable by other users for example if a user set up an OSC receiver with the port of 9000 another user will not be able to use that even if the port has changed

this means no one will be able to use the port of 9000 because it was already used by a previous user

### To Reproduce

1. Set up an OSC receiver with a port of 9000 make sure that you have working data with it
2. Change the port to something else let's say from 9000 to 8000
3. Set up another OSC receiver with the port of 9000 with someone else simulating it
4. you will observe that it won't work whatsoever until you edit the port from something else other than 9000

### Expected behavior

A port should become available once a user is not using it anymore

### Resonite Version Number

2024.4.30.495

### What Platforms does this occur on?

Windows

### What headset if any do you use?

Desktop

### Log Files

not available at the moment

### Reporters

@epicEaston197

**Comment by Frooxius** [comment]
I'm a bit confused by this. From the sound of it, it sounds like if one user uses a port, then another user cannot use the same port? 

Or is this for the same user? The replication steps seem to assume a single user doing these steps.

Also we'll need logs for this too.

**Comment by shiftyscales** [comment]
@epicEaston197 - This issue still needs logs provided.

**Comment by epicEaston197** [comment]
Yeah sorry I completely forgot about this issue because it's hard to replicate without someone else and I've not gotten the opportunity yet to test this

**Comment by shiftyscales** [comment]
Is this still an issue, @epicEaston197? If so, it still needs logs. Otherwise, it needs closed.

**Comment by epicEaston197** [comment]
I'm just going to close this issue i'm not going to bother doing much else if no one else has had this issue apparently if this issue comes up again it can either be reopened or someone can make a new one
